### PR TITLE
Move snyk-code scan command to Makefile

### PR DESCRIPTION
**Pull request by bear-redhat**
Move `snyk-code` scan command to `Makefile`

Depends on https://github.com/openshift/ci-tools/pull/2906

### CODE REVIEW

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-presubmits.yaml [hunk 1]
```diff
@@ -682,12 +682,9 @@ presubmits:
     spec:
       containers:
       - args:
-        - code
-        - test
-        - --project-name=ci-tools
-        - --org=red-hat-org
+        - analyse-code
```
[the remark this answers is not in the document]
Comment (bear-redhat) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-presubmits.yaml:685]: It's right here: https://github.com/openshift/ci-tools/pull/2906 :-D

**Comment by droslean** [comment]
/hold

Needs https://github.com/openshift/ci-tools/pull/2906

**Comment by droslean** [comment]
/lgtm

@bear-redhat Hold cancel once https://github.com/openshift/ci-tools/pull/2906 lands.

**Pull request merged**